plugins/salesforce/core/src/main/java/org/pentaho/di/trans/steps/salesforce/SalesforceStep.java

```diff
@@ -191,8 +192,7 @@ public boolean testConnection() {
         try {
           connection.close();
         } catch ( Exception e ) {
-          successConnection = false;
-          logError( e.getMessage() );
+          //Ignore
```

Comment:
Better to add a logger please.

Reply:
This is old code that's just moved out of SWT dialog class.